File: Training-Workspace/Web Development/Day_Four_Tasks/linearSearch.js
const prompt = require("prompt-sync")();

function linearSearch(arr, x) {
  let flag = true;
  for (let i = 0; i < arr.length; i++) {
    if (arr[i] === x) {
      console.log("Element Found At : " + (i + 1));
      flag = 0;
      break;
    }
  }
  if (flag) {
    console.log("Element Not Found!");
  }
}

var N = prompt("Enter The Array Size: ");
var array = [N];

for (let index = 0; index < N; index++) {
  array[index] = prompt("Enter Array Element " + (index + 1) + ": ");
}

var key = prompt("Enter The Key: ");

linearSearch(array, key);

File: Eclipse-Workspace/PLT_Exercises/src/LinearSearch.java
import java.util.*;

public class LinearSearch {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		System.out.print("Enter The Size Of The Array : ");
		int arraySize = sc.nextInt();
		int[] array = new int[arraySize];
		for (int i = 0; i < arraySize; i++) {
			array[i] = sc.nextInt();
		}
		System.out.print("Array Elements : " + Arrays.toString(array));
		System.out.println();
		System.out.print("Enter The Search Element : ");
		int searchElement = sc.nextInt();
		boolean flag = false;
		for (int i = 0; i < arraySize; i++) {
			if (array[i] == searchElement) {
				flag = true;
				break;
			}

		}
		if (flag)
			System.out.println(searchElement + " Is Found!");
		else
			System.out.println(searchElement + " Is Not Found!");

	}

}
